parsec/crypto.py

```diff
@@ -137,6 +159,11 @@ def generate(cls, *args, **kwargs) -> "SigningKey":
     def __eq__(self, other):
         return isinstance(other, _SigningKey) and self._signing_key == other._signing_key
 
+    # Quick fix, waiting for binding signingKey
```

Comment:
How's that a quick fix? Looks fine to me.

Reply:
cf: see the comment above

Reply:
I would also remove the "quick fix" comment given the change is fine (and saying quick fix imply something is fishy ^^)